test/integration/targets/vmware_guest/tasks/clone_set_manual_mac_address.yml

```diff
@@ -0,0 +1,43 @@
+- name: Power off clone source VMs
+  vmware_guest:
+    validate_certs: False
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    name: "{{ item.name }}"
+    datacenter: "{{ dc1 }}"
+    folder: "{{ item.folder }}"
+    state: poweredoff
+  loop: "{{ virtual_machines }}"
+
+- name: clone VM from template and set manual mac address
+  vmware_guest:
+    validate_certs: False
+    hostname: "{{ vcenter_hostname }}"
+    username: "{{ vcenter_username }}"
+    password: "{{ vcenter_password }}"
+    name: "{{ 'mac_manual_' + item.name }}"
+    template: "{{ item.name }}"
+    datacenter: "{{ dc1 }}"
+    state: poweredoff
+    folder: "{{ item.folder }}"
+    networks:
+        - name: VM Network
+          ip: "192.168.10.1{{ index }}"
+          netmask: 255.255.255.0
+          gateway: 192.168.10.254
+          mac: "aa:bb:cc:dd:aa:b{{ index }}"
+  loop: "{{ virtual_machines }}"
```

Comment:
Please avoid the loop and just use the first machine.

See: https://docs.ansible.com/ansible/devel/dev_guide/platforms/vmware_guidelines.html#no-need-to-create-too-much-resources